gui/storage/forms.py

```diff
@@ -130,6 +130,12 @@ def _clean_zfssize_fields(form, attrs, prefix):
     for attr in attrs:
         formfield = '%s%s' % (prefix, attr)
         match = r.match(cdata[formfield].replace(' ', ''))
+        try:
+            int(cdata[formfield])
+            continue
+        except ValueError:
+            pass
+
```

Comment:
Well, then it will skip all the rest, thats some another bug, can you leave it off and I'll see whats up with that?